linkedevents/settings.py

```diff
@@ -293,8 +292,6 @@ def get_git_revision_hash() -> str:
 LOGOUT_REDIRECT_URL = "/"
 ACCOUNT_LOGOUT_ON_GET = True
 
-# User will be assigned to a default organization if user doesn't have any
-ENABLE_USER_DEFAULT_ORGANIZATION = env("ENABLE_USER_DEFAULT_ORGANIZATION")
 # Default organization to which a user will be assigned if user doesn't
 # have any organization. Default organization is created if it doesn't exist.
 USER_DEFAULT_ORGANIZATION_ID = env("USER_DEFAULT_ORGANIZATION_ID")
```

Comment:
This comment should be slightly changed, since the default organization is not set for the user anymore. I wonder if the setting variable should also be changed to something more descriptive. :thinking: Some quickly made up alternatives: `EVENT_DEFAULT_ORGANIZATION`, `EXTERNAL_USER_EVENT_ORGANIZATION`